### PR TITLE
Entity lookup plugin. Failures generation fixes

### PR DESCRIPTION
**Pull request by mikhailmikhaylov**
I've rebased Entity Lookup Plugin in another branch to leave some working version. If everything is fine, this PR should override previous one.

I've changed failures generation to protobuf-desc-files-based

### CODE REVIEW

File: failures-gen-sample/generated/main/java/org/spine3/sample/failures/Failures.java [hunk 1]
```diff
@@ -983,14 +983,14 @@ public Failure2 parsePartialFrom(
       com.google.protobuf.MessageOrBuilder {
 
     /**
-     * <code>optional string id = 1;</code>
```
Comment (AlexanderLitus) [inline failures-gen-sample/generated/main/java/org/spine3/sample/failures/Failures.java]: Why this generated code is added to VCS?

File: failures-gen-sample/src/main/proto/spine/sample/failures/failures.proto [hunk 2]
```diff
@@ -43,3 +44,19 @@ message Failure3 {
...
+    enum Failure4InnerEnum {
+        OK = 0;
+    }
+
+    string id = 1;
+    google.protobuf.Any any_id = 2;
+    bool boolean_id = 3;
+    Failure4Inner inner_id = 4;
+    Failure4InnerEnum inner_enum_id = 5;
+}
```
Comment (AlexanderLitus) [inline failures-gen-sample/src/main/proto/spine/sample/failures/failures.proto]: EOF, please.

File: failures-gen-sample/generated/main/java/org/spine3/sample/failures/Failures.java (hunk 1 above)
Comment (mikhailmikhaylov) [inline failures-gen-sample/generated/main/java/org/spine3/sample/failures/Failures.java]: It's definitely in gitignore now, and the diff is outdated. I've also seen this file as "changed", but it should be excluded from repository already

**Comment by AlexanderLitus** [comment]
OTT, LGTM.

File: failures-gen-sample/src/main/proto/spine/sample/failures/failures.proto (hunk 2 above)
Comment (mikhailmikhaylov) [inline failures-gen-sample/src/main/proto/spine/sample/failures/failures.proto]: Done

**Pull request merged**